product_multi_image/views/product_template_view.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- © 2015 Antiun Ingeniería S.L. - Jairo Llopis
+     © 2014-2016 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl-3). -->
+<openerp>
+
+    <record id="view_product_template_form_img_inh" model="ir.ui.view">
+        <field name="name">Add multi images</field>
+        <field name="model">product.template</field>
+        <field name="inherit_id" ref="product.product_template_form_view"/>
+        <field name="arch" type="xml">
+            <xpath expr="/form/sheet/notebook" position="inside">
+                <page string="Images" name="multi_image">
+                    <field name="image_ids"
+                           nolabel="1"
+                           context="{
+                               'default_owner_model': 'product.template',
+                               'default_owner_id': id,
+                               'form_view_ref': 'product_multi_image.image_form_view',
+                               'kanban_view_ref': 'product_multi_image.image_kanban_view',
+                           }"
+			   mode="kanban" />
+                    </field>
```

Comment:
```suggestion    
```
This </field> tag is unnecessary, the field name="image_ids" id defined with an autoclosing field tag.

Reply:
@qtheuret I confirm the suggestion of @hugosantosred And it make the module uninstallable.
Are you using the module?
 Could you commit the suggestion made by @hugosantosred  ?